tools/spec-gen-sdk/src/automation/workflowPackage.ts

```diff
@@ -155,7 +155,10 @@ const workflowPkgSaveApiViewArtifact = async (context: WorkflowContext, pkg: Pac
     return;
   }
 
-  const destination = path.join(context.config.workingFolder, 'out', 'stagedArtifacts', pkg.name);
+  const stagedArtifactsFolder = path.join(context.config.workingFolder, 'out', 'stagedArtifacts');
```

Comment:
Could you move line#132-#133 to the place before #line#127 then we don't need to set again here? 

As we just synced in standup, you will copy the artifact for Go language too, right?

Reply:
So it turns out I actually don't need to copy it. At least as far as APIView is concerned it is already copied. I just need to make sure the variable for the destination directory is set.

Reply:
ok. Do you need to make another PR for other changes in spec-gen-sdk tool? Otherwise, you should bump package version and update changelog for a new release.